app/controllers/dashboards_controller.rb

```diff
@@ -24,7 +25,7 @@ def happy_things_by_period(period, friend_ids)
 
   def fetch_daily_quote
     quote_service = QuoteService.new('happiness')
-    quote_info = quote_service.call
+    quote_service.call
```

Comment:
What happens if the service is down? Would we get an error trying to render the view?

Reply:
I added a fetch_fake_quote in the call too 👻